### ISSUE TITLE
Update Plain Language Impact Expo ppt

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/onrr.gov-site#2690]
Add slide to discuss reading level detection software

**Comment by cthomasONRR** [comment]
@Maroyafaied The updated presentation is ready for your review. It's located [here](https://doimspp.sharepoint.com/:f:/r/sites/NRRD/Shared%20Documents/General/Outreach/Proposals%20%26%20Presentations/FY23/ONRR%20Impact%20Expo%202023?csf=1&web=1&e=EkO2og).

I added slides 11, 14, and 15. I also updated the demo page to be our plain language webpage (https://www.onrr.gov/government-initiatives-laws?tabs=plain-language). I felt weird/bad critiquing a webpage that wasn't ours.

**Comment by Maroyafaied** [comment]
@cthomasONRR I reviewed and left one comment within the ppt, and I'll leave my other comment here since it's long.  Overall , the new slides are great and not overwhelming. My comment is I expect someone to ask what grade level should they aim for and what they grade levels mean. Here's a good answer I found on hemmingway and a chart explaining that I found somewhere else, you don't have to add this info in the slides but just wanted to provide it as a reference. 

"Now, when we say “grade level,” we aren't saying that's who you're writing for. In fact, Ernest Hemingway's work scores as low 5th grade, despite his adult audience. What our measurement actually gauges is the lowest education needed to understand your prose. Studies have shown the average American reads at a tenth-grade level — so that's a good target."

![image](https://github.com/DOI-ONRR/onrr.gov-site/assets/40175688/9455b8e0-50ca-4b42-9c94-f65753102f30)
